.circleci/config.yml

```diff
@@ -24,14 +24,14 @@ jobs:
           filename: combined-package-lock.txt
       - restore_cache:
           keys:
-            - deps-v3-{{ .Branch }}-{{ checksum "combined-package-lock.txt" }}
-            - deps-v3-{{ .Branch }}
-            - deps-v3-
+            - deps-v4-{{ .Branch }}-{{ checksum "combined-package-lock.txt" }}
+            - deps-v4-{{ .Branch }}
+            - deps-v4-
```

Comment:
@rebelchris I have updated the deps reference for the keys. Am I missing something?

Reply:
Nope it should be just the keys, I had this before for a update, never got that to work.
Perhaps we can leave this part and see what happens when we merge into companion branch?